### PR TITLE
Game communities

### PR DESCRIPTION
**Pull request by nilllzz**

### CODE REVIEW

File: components/community/thumbnail/img/img.ts [hunk 1]
```diff
@@ -12,11 +12,6 @@ export default class AppCommunityThumbnailImg extends Vue {
...
 
-	get src() {
-		if (!this.community.thumbnail) {
-			return '';
-		}
-
-		return this.community.thumbnail.mediaserver_url;
-	}
+	@Prop(Boolean)
+	isEditing!: boolean;
```
Comment (YLivay) [inline components/community/thumbnail/img/img.ts]: Remove unused prop

File: components/content/content-editor/content-editor.ts [hunk 2]
```diff
@@ -339,6 +340,11 @@ export default class AppContentEditor extends Vue implements ContentOwner {
 		}
 	}
 
+	public updateIsEmpty(state: EditorState) {
+		// The "empty" prosemirror document takes up a length of 4.
+		this.isEmpty = state.doc.nodeSize <= 4;
```
Comment (YLivay) [inline components/content/content-editor/content-editor.ts:345]: can a super short doc take up less than 4 somehow?

File: components/content/content-editor/content-editor.ts [hunk 3]
```diff
@@ -253,6 +253,7 @@ export default class AppContentEditor extends Vue implements ContentOwner {
 				'data-prevent-shortkey': '',
 			},
 		});
+		this.updateIsEmpty(state);
```
Comment (YLivay) [inline components/content/content-editor/content-editor.ts:256]: this is that unrelated fix you cherry picked to fix the placeholder showing up in game descriptions on load?

File: components/notification/notification-model.ts [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import { Community } from 'game-jolt-frontend-lib/components/community/community.model';
```
Comment (YLivay) [inline components/notification/notification-model.ts]: Don't use `game-jolt-frontend-lib` in this repo

File: components/community/community.model.ts [hunk 5]
```diff
@@ -1,9 +1,11 @@
+import { CommunityTag } from 'game-jolt-frontend-lib/components/community/tag/tag.model';
```
Comment (YLivay) [inline components/community/community.model.ts]: Don't use `game-jolt-frontend-lib` in this repo.

**Review by YLivay — CHANGES_REQUESTED** [review]

File: components/content/content-editor/content-editor.ts (hunk 2 above)
Comment (nilllzz) [reply components/content/content-editor/content-editor.ts:345]: No, 4 is the default size for no content at all (doc and paragraph opening and closing tags).
This is not the text length or even content length, it's prosemirror's calculated length for their internal doc thing.

File: components/content/content-editor/content-editor.ts (hunk 3 above)
Comment (nilllzz) [reply components/content/content-editor/content-editor.ts:256]: yep

**Review by YLivay — APPROVED** [review]
